content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,354 @@
+
+```css
+#container{
+    min-width: 310px; 
+    height: 400px; 
+    margin: 0 auto
+}
+```
+
+The `html` code is shown below:
+
+```html
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <title>Piechart</title>
+
+        <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
+
+        <script src="./node_modules/highcharts/highcharts.js"></script>
+        <script src="exporting.js"></script>
+
+        <script type="text/javascript" src="./node_modules/brython/brython.js"></script>
+        <script src="./node_modules/brython/brython_stdlib.js"></script>
+        <script type="text/python3" src="chart.py"></script>
+
+    </head>
+    <body onload="brython(1)">
+
+        <div id="container" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
+
+    </body>
+</html>
+```
+
+##### Chart.py file 
+The chart.py file contains the following code:
+
+```python
+from browser import window
+
+b_highchart = window.Highcharts.Chart.new
+
+b_highchart(
+    {
+        "chart": {
+            "plotBackgroundColor": "#FCFFC5",
+            "plotBorderWidth": None,
+            "plotShadow": False,
+            "renderTo": "container",
+        },
+        "title": {"text": "Religious Distribution in Nigeria"},
+        "tooltip": {"pointFormat": "<b>{point.percentage:.1f}%</b>"},
+        "plotOptions": {
+            "pie": {
+                "allowPointSelect": True,
+                "cursor": "pointer",
+                "dataLabels": {
+                    "enabled": True,
+                    "format": "<b>{point.name}</b>: {point.percentage:.1f} %",
+                    "style": {"color": "black"},
+                },
+            }
+        },
+        "series": [
+            {
+                "type": "pie",
+                "data": [
+                    ["Indigenious Beliefs", 10.7],
+                    ["Other Christian", 35.3],
+                    {
+                        "name": "Roman Catholicism",
+                        "y": 10.0,
+                        "sliced": True,
+                        "selected": True,
+                    },
+                    ["Islam", 43.5],
+                    ["Others", 0.5],
+                ],
+            }
+        ],
+    }
+)
+```
+
+The code above is used to plot the pie chart using the following methods:
+- **`plotBackgroundColor`** 
+  It is used to plot the background color for the chart area.
+
+- **`plotBorderWidth`** 
+  This is used to plot the width of the chart area in pixels.
+
+- **`plotShadow`**  
+  This method takes in boolean values. It adds a drop shadow to the area. 
+
+- **`renderTo`**
+  This method is similar to `document.getElementBy` in javascript. It will take the `id` of the `div` tag in our HTML code.
+
+- **`tooltip`**
+  This object contains the details in the small box when we hover on our chart.
+
```

Comment:
```suggestion
The code above is used to plot the pie chart using the following methods:
- `plotBackgroundColor`
It is used to plot the background color for the chart area.

- `plotBorderWidth`
It is used to plot the width of the chart area in pixels.

- `plotShadow`
This method takes in boolean values. It adds a drop shadow to the area. 

- `renderTo`
This method is similar to `document.getElementBy` in javascript. It will take the `id` of the `div` tag in our HTML code.

- `tooltip`
This object contains the details in the small box when we hover on our chart.

```